src/jupyter_contrib_nbextensions/nbextensions/go_to_current_running_cell/README.md

```diff
@@ -0,0 +1,19 @@
+# Go to Running Cell
+
+This is an extension allows you to jump to the current running cell. You can also activate this functionality automatically, i.e., your view is always scolling to the current cell.
+
+Button: A button with eye icon that you can go to the first running cell.  
+![button](anchor.png)
+
+# Keyboard shortcuts:   
```

Comment:
Could you make this & the demo heading level 2 (rather than 1)? It helps the contents for the auto-generated docs not get mangled :)

Reply:
updated